ngraph/test/backend/gather.in.cpp

```diff
@@ -324,313 +324,6 @@ NGRAPH_TEST(${BACKEND_NAME}, gather_scalar_indices_axis_1_2d_input)
         (vector<float>{1.0f, 2.0f, 3.0f}), read_vector<float>(result), MIN_FLOAT_TOLERANCE_BITS));
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, gather_nd_single_indices)
-{
-    Shape params_shape{3, 3};
-    Shape indices_shape{2};
-    Shape out_shape{};
-    auto P = make_shared<op::Parameter>(element::f32, params_shape);
-    auto I = make_shared<op::Parameter>(element::i32, indices_shape);
-    auto G = make_shared<op::GatherND>(P, I);
```

Comment:
I'm confused. What is wrong with GatherND?

Reply:
Yeah, I'm also confused about this tests disabled, and that's what I'm referring to in the  "Check GatherND tests deletion." for me and @iefode

Reply:
Fixed

Reply:
Tests deletion reverted